debian/control

```diff
@@ -29,7 +29,7 @@ Build-Depends: debhelper (>= 10),
                libudev-dev [linux-any],
                gobject-introspection (>= 0.10.2-1~),
                libgirepository1.0-dev (>= 0.10.2-1~),
-               gsettings-desktop-schemas-dev (>= 3.5.91),
+               gsettings-desktop-schemas-dev (>= 3.24.0),
```

Comment:
The version should be `(>= 3.24.0-2endless1)`, since that's where the new API is added. (Multiple places in the PR.)

Reply:
Good point